### PR TITLE
chore(eslintrc): update .eslintrc.json

### LINKED ISSUE #58: ESLint configuration is invalid: Unexpected top-level property "ecmaFeatures".
**I'm submitting a bug report**

* **Library Version:**
1.0.0

**Please tell us about your environment:**
* **Operating System:**
Linux (Xubuntu 16.04)

* **Node Version:**
8.1.0

* **NPM Version:**
5.0.3

* **Browser:**
all?

* **Language:**
ESNext

**Current behavior:**
the .eslintrc.json file is only valid for eslint <2.0.0, and as such the linter fails to do it's job.

acoording to the eslint [docs](http://eslint.org/docs/user-guide/migrating-to-2.0.0#language-options)
>Prior to 2.0.0, the way to enable language options was by using ecmaFeatures in your configuration. In 2.0.0:

>* The ecmaFeatures property is now under a top-level parserOptions property.
if you run the linter you get an error:

error screenshot

and the .eslintrc.json just references aurelia-tools
```json
{
  "extends": "./node_modules/aurelia-tools/.eslintrc.json"
}
```

**Expected/desired behavior:**

* **What is the expected behavior?**
that the .eslintrc.json be updated to work with the latest version of eslint. Our project is based on one of the ESNext skeletons, and their .eslintrc.json reference aurelia-tools' .eslintrc.json

* **What is the motivation / use case for changing the behavior?**
We need good tooling as part of "aurelia-tools"

### PR DESCRIPTION
**Pull request by obedm503**
- fixed .eslintrc.json to conform to eslint@4.x format
- added eslint and babel-eslint as devDependencies
- npm automatically created package-lock.json

Closes #58

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,5 +20,9 @@
   "dependencies": {
     "breeze-dag": "^0.1.0",
     "through2": "^2.0.0"
+  },
+  "devDependencies": {
```
Comment (gheoan) [inline package.json]: Shouldn't these two be peerDependencies instead, because they are installed by the aurelia skeleton?

Comment (obedm503) [reply package.json]: Yes. I updated the file

**Comment by obedm503** [comment]
anything here @EisenbergEffect ?

**Comment by EisenbergEffect** [comment]
@obedm503 Am I correct in understanding that we'd need to update all our libraries for this to function correctly?

**Comment by obedm503** [comment]
all the libraries reference this same file, you would just have to update the eslint version to the one used here

**Review by niieani — APPROVED** [review]
Looks good to me @EisenbergEffect.

**Comment by EisenbergEffect** [comment]
@niieani Am I correct in understanding that this is a breaking change? So, we need to bump tools to 2.0, yes?

**Comment by niieani** [comment]
@EisenbergEffect yeah, it would be safe to do so.

**Comment by pkkummermo** [comment]
Any plans to merge this one?:) I'm getting tired of VS Code ranting ^^

**Comment by pkkummermo** [comment]
Thanks! :tada:

**Pull request merged**